src/Symfony/Component/Notifier/Bridge/AllMySms/AllMySmsTransport.php

```diff
@@ -28,16 +28,13 @@ final class AllMySmsTransport extends AbstractTransport
 {
     protected const HOST = 'api.allmysms.com';
 
-    private string $login;
-    private string $apiKey;
-    private ?string $from;
-
-    public function __construct(string $login, #[\SensitiveParameter] string $apiKey, string $from = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
-    {
-        $this->login = $login;
-        $this->apiKey = $apiKey;
-        $this->from = $from;
-
+    public function __construct(
+        private string $login,
+        #[\SensitiveParameter] private string $apiKey,
+        private ?string $from = null,
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null
```

Comment:
please check missing trailing commas in the PR

Reply:
Isn't there a CS-Fixer rule to ensure this? 🤔 

cc @keradus

Reply:
Done

Reply:
not sure if you were aware what you asked for... but here we go #53352